### DISCUSSION TITLE
Using CSLA4 Book Samples

### DISCUSSION THREAD

**Discussion by chatzikk** [MarimerLLC/csla#1671] [Questions] [answered]
I want to download the CSLA4 Book Samples. 
The last time I downloaded them from address: http://download.lhotka.net/
Does anybody know where can I find them?

**Comment by danielmartind** [comment]
There are different branches for the older versions you get the samples from or also on the releases page. Also [https://cslanet.com/Download.html](https://cslanet.com/Download.html)

**Comment by chatzikk** [comment]
Thank you very much for your answer but I am looking for the samples that are included with the e-books

**Reply by danielmartind** [reply]
Sorry, thought the samples were using the project tracker app. I had forgotten there were other samples with the books.

**Comment by rockfordlhotka** [comment, marked as answer]
@chatzikk I replaced the old store and download mechanism a few years ago. The book content and samples are on the new store, and I don't think I have the code samples anywhere else at this time.

Can you do the following?

1. Create an account on https://store.lhotka.net
1. Email me - rocky at lhotka dot net - so I have your email address

I will grant your store account access to the content.
